pdf/config.json5

```diff
@@ -504,6 +682,34 @@
                 type: 'regex-counter',
                 stdout: {},
                 stderr: {
+                    '^\\s*(error: cannot create appearance stream).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(error: unknown colorspace).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(error: zlib error: invalid).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(warning: cannot create ToUnicode mapping).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(warning: ignoring one to many mapping in cmap).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(warning: non-embedded font using identity encoding).*$': {
+                        nameGroup: 1
+                    },
+
+                    '^\\s*(warning: non-page object in page tree).*$': {
+                        nameGroup: 1
+                    },
+
```

Comment:
The duplication of all these is a bit unfortunate, but at the end of the day, it probably isn't a big enough issue to merit a larger fix.